_episodes/11-obsolete-gitlab.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
NOTE: This lesson is obsolete (hence the re-naming). keeping it for posterity, but it is no longer relevant. No other content has been changed